app/params/weights.go

```diff
@@ -1,16 +1,17 @@
-package app
+package params
 
 // Default simulation operation weights for messages and gov proposals.
 const (
 	DefaultWeightMsgLiquidStake                 int = 50
-	DefaultWeightMsgLiquidUnstake               int = 20
+	DefaultWeightMsgLiquidUnstake               int = 40
 	DefaultWeightMsgProvideInsurance            int = 70
 	DefaultWeightMsgCancelProvideInsurance      int = 10
 	DefaultWeightMsgDepositInsurance            int = 10
 	DefaultWeightMsgWithdrawInsurance           int = 20
 	DefaultWeightMsgWithdrawInsuranceCommission int = 10
-	DefaultWeightMsgClaimDiscountedReward       int = 10
+	DefaultWeightMsgClaimDiscountedReward       int = 20
 
 	DefaultWeightUpdateDynamicFeeRateProposal int = 2
 	DefaultWeightUpdateMaximumDiscountRate        = 2
+	DefaultWeightAdvanceEpoch                 int = 5
```

Comment:
Unused weight variables

Reply:
done in https://github.com/b-harvest/Canto/commit/a9b210366eed46d0c3a9baaaa2a580bb8199b8eb